app/components/map/index.js

```diff
@@ -148,12 +144,11 @@ class Map extends Component {
       !isEqual(nextProps.areaCoordinates, this.props.areaCoordinates),
       !isEqual(nextProps.area, this.props.area),
       nextProps.canDisplayAlerts !== this.props.canDisplayAlerts,
-      !isEqual(nextProps.center, this.props.center),
       !isEqual(nextProps.contextualLayer, this.props.contextualLayer),
       !isEqual(nextState.lastPosition, this.state.lastPosition),
       nextState.hasCompass !== this.state.hasCompass,
       nextState.heading !== this.state.heading,
-      !isEqual(nextState.region, this.state.region),
+      // !isEqual(nextState.region, this.state.region),
```

Comment:
if you don't use it, you lose it. 😛

Reply:
But we aren't using it on the map so it is working perfectly and we are avoiding re-renders

Reply:
I mean we can remove the commented line